lib/rules/function-no-unknown/index.mjs

```diff
@@ -67,14 +67,18 @@ const rule = (primary, secondaryOptions) => {
 
 				if (FUNCTIONS.includes(name.toLowerCase())) return;
 
+				const declIndex = declarationValueIndex(decl);
+				const index = declIndex + node.name[2];
+				const endIndex = declIndex + node.name[3];
```

Comment:
note

`2` is "Start position of representation"
`3` is "End position of representation"